package.json

```diff
@@ -31,13 +31,13 @@
     "underscore": "^1.7.0",
     "body-parser": "^1.9.0",
     "validator": "^3.21.0",
-    "string": "^2.2.0"
+    "string": "^2.2.0",
+    "grunt-cli": "~0.1.13"
```

Comment:
I moved it because when using grunt I was getting an annoying warning about grunt-cli not being correctly installed locally. Did a bit of research and turned out that was a possible reason, and in fact by doing that the warning disappeared.
Think I can replicate the problem at any time and show you - or just move it back to DevDependencies.